bigtable/CMakeLists.txt

```diff
@@ -122,8 +125,79 @@ target_link_libraries(googleapis ${GRPCPP_LIBRARIES} ${GRPC_LIBRARIES} ${PROTOBU
 # Enable unit tests
 enable_testing()
 
+# This is an extremely ugly hack to link with Abseil code
+# TODO(dmahu): CMakeLists need refactoring to allow compiling Abseil
+add_library(hack_abseil
```

Comment:
Can we use GUARDED_BY() and friends with a regular `std::mutex`?  If so, we could eliminate this hack, right?  Even if we couldn't, I would prefer to eliminate the hack and use plain `std::mutex`.

Reply:
Turns out we can use annotations with std::mutex.